frontend/packages/kubevirt-plugin/src/selectors/nad/combined.ts

```diff
@@ -3,19 +3,13 @@ import { getName } from '@console/shared/src';
 import { NetworkType } from '../../constants/vm';
 import { NetworkWrapper } from '../../k8s/wrapper/vm/network-wrapper';
 
-export const getNetworkChoices = (
-  nads: K8sResourceKind[],
-  usedNetworkNames: Set<string>,
-  allowPodNetwork,
-): NetworkWrapper[] => {
-  const networkChoices = nads
-    .map((nad) => getName(nad))
-    .filter((nadName) => !(usedNetworkNames && usedNetworkNames.has(nadName)))
-    .map((networkName) =>
-      new NetworkWrapper().setType(NetworkType.MULTUS, {
-        networkName,
-      }),
-    );
+export const getNetworkChoices = (nads: K8sResourceKind[], allowPodNetwork): NetworkWrapper[] => {
```

Comment:
IMO it doesn't make sense to expose this function as public anymore. Because it is used only by nic-modal and has a special syntax (allowPodNetwork, return Networks without name).

Can you please move it to `nic-modal.tsx` or inline it in Network component?

Reply:
Done!